.travis.yml

```diff
@@ -75,8 +72,9 @@ install:
 
 before_script:
 #  # Start WPS service on port 5000 on 0.0.0.0
-#  - raven start --daemon --bind-host 0.0.0.0 --port 5000
-  - bash -c "source $HOME/miniconda3/bin/activate raven && make start"
+    #  - raven start --daemon --bind-host 0.0.0.0 --port 5000
+    - bash -c "source $HOME/miniconda3/bin/activate raven && pip install git+https://github.com/pydata/xarray.git@master"
```

Comment:
Is this install from xarray.git@master still necessary with xarray 0.16 from `master` branch? @Zeitsperre  (a125ff4bdd86881239226ac77dcf54ee721ad8b1)

Reply:
With xarray at 0.16, the sdba functions that Pascal was working on are integrated, so no. This workaround can blissfully be thrown in the trash.